scripts/ci/node_package.sh

```diff
@@ -7,31 +7,31 @@ echo "node version is:"
 which node
 node -v
 
-if [[ ${PUBLISH} == 'On' ]]; then
-    echo "PUBLISH is set to '${PUBLISH}', publishing!"
-    NPM_FLAGS=''
-    if [[ ${BUILD_TYPE} == "Debug" ]]; then
-        NPM_FLAGS='--debug'
-    fi
+NPM_FLAGS=''
```

Comment:
I don't see why don't we run it on every CI. We have a couple of useful checks here and it turned out that one of them was red for me.

Reply:
> one of them was red

I mean this check on Linux that we have proper ORIGIN flags

Reply:
Yep, makes sense.